include/nccl_ofi_param.h

```diff
@@ -338,6 +338,37 @@ OFI_NCCL_PARAM_INT(errorcheck_mutex, "ERRORCHECK_MUTEX",
  */
 OFI_NCCL_PARAM_INT(endpoint_per_communicator, "ENDPOINT_PER_COMM", 0);
 
+/*
+ * Force using a specific tuner type.
+ * "Internal" for NCCL internal tuner.
+ * "Region" for NCCL OFI Region base tuner.
+ * "Model" for NCCL OFI Model base tuner.
+ */
+OFI_NCCL_PARAM_STR(tuner_force_type, "TUNER_FORCE_TYPE", NULL);
```

Comment:
Nit, s/TUNER_FORCE_TYPE/TUNER_TYPE/.. since by setting an env we're overriding behavior anyway.

Reply:
ack